Accessibility/accessibility_tools.md

```diff
@@ -0,0 +1,23 @@
+### Tools to test accessibility
+* [Axe-devtools](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US)
+
+* [axe-core-gems](https://github.com/dequelabs/axe-core-gems)
+
+* [Wave](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh?hl=en-US)
+
+* [Lighthouse](https://developer.chrome.com/blog/lighthouse-load-performance/)
+
+* [Pa11y](https://www.npmjs.com/package/pa11y)
+
+* Screen reader:   
```

Comment:
Add a link to a new md file with the screen reader commands that are most useful for evaluation.